app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  before_action :require_user
+
+  def create
+    @like = Like.new(recipe_id: params[:recipe_id])
+    @recipe = Recipe.find(params[:recipe_id])
```

Comment:
why is there a @recipe var here? Is it being used?

Reply:
Solved in the nexts comments!